.travis.yml

```diff
@@ -4,9 +4,25 @@ php:
   - 5.4
   - 5.5
 
+env:
+  - SYMFONY_VERSION="2.3.*"
+  - SYMFONY_VERSION="2.6.*"
+  - SYMFONY_VERSION="2.7.*"
+  - SYMFONY_VERSION="2.8.x-dev"
+  - SYMFONY_VERSION="3.0.x-dev"
```

Comment:
Please don't test all Symfony versions on all. This is a waste of Travis resources